.github/workflows/ci-build.yaml

```diff
@@ -0,0 +1,92 @@
+name: CI
+
+on:
+  # Trigger the workflow on push or pull request,
+  # only on the master or jax branches
+  push:
+    branches:
```

Comment:
This is fine for now but we might find it easier to trigger the workflows on work-in-progress branches as well. Can a workflow be manually triggered as well?

Reply:
I believe it is now set up to be manually triggered - however we won't see the button until the workflow file is merged into master